tests/test_cmdline.py

```diff
@@ -86,7 +86,7 @@ def test_help(self):
     # FIXME: More extensive tests
 
 
-class StatusTestCase(TestCase):
+class StatusTestCase:
```

Comment:
I think I missed this, although I thought I had pulled the latest version on which I made a branch.